nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -61,6 +73,7 @@ let
       #Let's hope 15M is enough
       dd bs=2048 count=7680 if=/dev/zero of="$out"
```

Comment:
What limit would you suggest? 20M? 50M?

Or should we calculate what is needed to fit ${efiDir} in there?